rero_ils/modules/documents/serializers/ris.py

```diff
@@ -65,6 +78,9 @@ def serialize_search(self, pid_fetcher, search_result, links=None,
 class RISFormatter(BaseDocumentFormatterMixin):
     """RIS formatter class."""
 
+    # RIS separator between key and value
+    separator = '  - '
```

Comment:
really ? 2 spaces before the dash ?

Reply:
yes. see https://fr.wikipedia.org/wiki/RIS_(format_de_fichier)#:~:text=Une%20balise%20est%20compos%C3%A9e%20de%20deux%20lettres%2C%20suivies%20de%20deux%20espaces%20et%20d%27un%20tiret%C2%A0%3A%20%C2%AB%C2%A0TY%20%2D%C2%A0%C2%BB